compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -247,43 +256,17 @@ internal class ComposeContainer(
         super.didReceiveMemoryWarning()
     }
 
-    @OptIn(ExperimentalComposeApi::class)
-    private fun setContent(content: @Composable () -> Unit) {
-        if (composeSceneMediator != null) {
-            return // already attached
-        }
-
-        val mediator = if (configuration.platformLayers) {
-            createSingleLayerComposeSceneMediator()
-        } else {
-            createMultiLayerComposeSceneMediator()
-        }
-        composeSceneMediator = mediator
-        mediator.setContent {
-            ProvideContainerCompositionLocals(this) {
-                content()
-            }
-        }
-        mediator.setLayout(SceneLayout.UseConstraintsToFillContainer)
-    }
-
-    private fun dispose() {
-        composeSceneMediator?.dispose()
-        composeSceneMediator = null
-        layers.fastForEach {
-            it.close()
-        }
+    fun createComposeSceneContext(platformContext: PlatformContext): ComposeSceneContext =
+        ComposeSceneContextImpl(platformContext)
 
-    }
+    private fun createSkikoUIView(mediator: ComposeSceneMediator):SkikoUIView = SkikoUIView(
```

Comment:
Mediator receives this function as a construction argument to pass itself to be fed its own `renderDelegate`.
Perhaps a `mediator` should only receive `transparency` parameter and create `SkikoUIView` on his own, and `renderDelegate` should be a private property?

Reply:
@MatkovIvan asked to create builder of SkikoUIView outside of ComposeSceneMediator.
This needs for better consistency with desired architecture and be simillar to Desktop implementation.
But, maybe we can simplify `renderDelegate`

Reply:
This discussion was resolved on meeting